tools/pd-simulator/main.go

```diff
@@ -43,7 +43,7 @@ import (
 )
 
 var (
-	pdAddr                      = flag.String("pd", "", "pd address")
+	pdAddr                      = flag.String("pd-endpoints", "", "pd address")
```

Comment:
Need to change `Readme.md`

Reply:
- change because https://github.com/tikv/tikv/blob/master/README.md#deploy-a-playground-with-binary
When deploy with `pd-endpoints`
- tiup changed also https://github.com/pingcap/tiup/pull/2162/files#diff-0e2023f88922646c56345bb747099b3171db8e56fc6d8cba07bdeeeeaf08064eR68